src/main/java/no/nav/familie/integrasjoner/dokdist/DokdistController.kt

```diff
@@ -24,7 +24,13 @@ class DokdistController(private val dokdistService: DokdistService) {
     @ExceptionHandler(HttpClientErrorException::class)
     fun handleHttpClientException(e: HttpClientErrorException): ResponseEntity<Ressurs<Any>> {
         secureLogger.warn("Feil ved distribusjon: ${e.message}")
-        return ResponseEntity.status(e.rawStatusCode).body(Ressurs.failure(e.message, error = e))
+        val ressurs: Ressurs<Any> = Ressurs(
+            data = e.responseBodyAsString,
+            status = Ressurs.Status.FEILET,
+            melding = e.message.toString(),
```

Comment:
Hvis message er null, så returneres "null" fra `toString`, er det kanskje bedre å returnere en tekst? 
```suggestion
            melding = e.message ?: "Uventet feil status=${e.rawStatusCode}",
```